### ISSUE TITLE
Can't log in, reset etc... pretty much stuck

### ISSUE THREAD

**Issue by crypt0-fusi0n** [keybase/keybase-issues#3396]
1. Downloaded win client
2. Signed up (set name for win device)
3. Set passphrase
4. Tried to edit device name. Couldn't find where so deleted device.
5. Now, can't log in via Windows client... ERROR: Cannot add a new device when all existing devices are revoked. Reset your account on keybase.io. Error code 218: GENERIC in method keybase.1.login.login
6. Went to keybase.io, can't find where to reset account.
7. Found Edit account details...
8. Get msg "Your account is in lockdown mode. You can change your e-mail or password in the Keybase client."
9. So basically stuck.

Bottom line, can still log into the website but not the windows client.

Thanks.

**Comment by maxtaco** [comment]
Email max@keyba.se to request an account reset.

**Comment by crypt0-fusi0n** [comment]
Hi Max,

Could you please reset my account?

Username: fusi0n

Thanks.

**Comment by maxtaco** [comment]
@zapu this user wound up in lockdown mode, remind me how? now the account cannot be reset.
